python-sdk/tutorials/automl-with-azureml/forecasting-pipelines/auto-ml-forecasting-pipelines.ipynb

```diff
@@ -0,0 +1,783 @@
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "inference_published_pipeline = inference_pipeline.publish(\n",
+    "    name=\"OJ Inference Test\", description=\"OJ Inference Test\"\n",
+    ")\n",
+    "print(\"Newly published pipeline id: {}\".format(inference_published_pipeline.id))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If `test_dataset` is going to refresh every 20 weeks before Friday 16:00 and we want to predict every 20 weeks (forecast_horizon), we can schedule our pipeline to run everyday at 16:00 to get daily inference results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# schedule\n",
+    "\n",
+    "from azureml.pipeline.core.schedule import ScheduleRecurrence, Schedule\n",
+    "\n",
+    "recurrence = ScheduleRecurrence(\n",
+    "    frequency=\"Week\", interval=20, week_days=[\"Friday\"], hours=[16], minutes=[0]\n",
+    ")\n",
+    "\n",
+    "schedule = Schedule.create(\n",
+    "    workspace=ws,\n",
+    "    name=\"OJ_Inference_schedule\",\n",
+    "    pipeline_id=inference_published_pipeline.id,\n",
+    "    experiment_name=\"Schedule-run-OJ\",\n",
+    "    recurrence=recurrence,\n",
+    "    wait_for_provisioning=True,\n",
+    "    description=\"Schedule Run\",\n",
+    ")\n",
+    "\n",
+    "# You may want to make sure that the schedule is provisioned properly\n",
+    "# before making any further changes to the schedule\n",
+    "\n",
+    "print(\"Created schedule with id: {}\".format(schedule.id))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### [Optional] Disable schedule"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "schedule.disable()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "authors": [
+   {
+    "name": "jialiu"
+   }
+  ],
+  "category": "tutorial",
+  "celltoolbar": "Raw Cell Format",
+  "compute": [
+   "Remote"
+  ],
+  "datasets": [
+   "Orange Juice Sales"
+  ],
+  "deployment": [
+   "Azure Container Instance"
+  ],
+  "exclude_from_index": false,
+  "framework": [
+   "Azure ML AutoML"
+  ],
+  "friendly_name": "Forecasting orange juice sales with deployment",
+  "index_order": 1,
+  "kernelspec": {
+   "display_name": "Python 3.6 - AzureML",
+   "language": "python",
+   "name": "python3-azureml"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.6.9"
+  },
+  "tags": [
+   "None"
+  ],
+  "task": "Forecasting"
+ },
+ "nbformat": 4,
```

Comment:
Should we demonstrate how we can download the data we have saved during prediction?

Reply:
The data is already saved in the tabular dataset format. The customer can take that from a start point.

Reply:
This is already available in retrieving the TabularDataset.